streams/examples/src/main/java/org/apache/kafka/streams/examples/pageview/PageViewTypedJob.java

```diff
@@ -124,9 +112,9 @@ public static void main(String[] args) throws Exception {
         serdeProps.put("JsonPOJOClass", RegionCount.class);
         regionCountSerializer.configure(serdeProps, false);
 
-        KStream<String, PageView> views = builder.stream(stringDeserializer, pageViewDeserializer, "streams-pageview-input");
+        KStream<String, PageView> views = builder.stream(Serdes.String(), Serdes.serdeFrom(null, pageViewDeserializer), "streams-pageview-input");
```

Comment:
@guozhang: Perhaps we should still enforce that users must always provide both ser and deser functionality?  I am torn on this one (that is, 1. supporting both "partial" serdes that you can't tell apart -- type-wise -- from "full" serdes, all of them implementing `Serde<T>` vs. 2. forcing users to always provide full serdes and throwing IAE if they attempt to pass in `null`), but I do feel that passing around `null` values isn't going to cut it, either.